app/workers/messaging.php

```diff
@@ -20,59 +30,180 @@
 
 class MessagingV1 extends Worker
 {
-    protected ?Adapter $sms = null;
-    protected ?string $from = null;
+    protected ?SMSAdapter $sms = null;
+    protected ?PushAdapter $push = null;
+    protected ?EmailAdapter $email = null;
+
+    protected ?Database $dbForProject = null;
+
 
     public function getName(): string
     {
         return "mails";
     }
 
-    public function init(): void
+    public function sms($record): ?SMSAdapter
```

Comment:
Let's move these functions after `processMessage` and make them private

Reply:
done this